src/cursive.rs

```diff
@@ -15,6 +15,22 @@ use views::{self, LayerPosition};
 /// Identifies a screen in the cursive root.
 pub type ScreenId = usize;
 
+/// Asynchronous callback function trait.
+///
+/// Every FnOnce(&mut Cursive) -> () + Send automatically
+/// implements this. This is a workaround only because
+/// Box<FnOnce()> is not working and FnBox is unstable.
+pub trait CbFunc: Send {
```

Comment:
Thanks! I think I got disapointed by the `FnBox` situation and just stopped thinking about it, without realizing a workaround like this could work. Awesome!